package.json

```diff
@@ -7,7 +7,6 @@
     "test": "jest --coverage",
     "test:dev": "jest --coverage --watch",
     "start": "tsc -p . --watch & nodemon build/app.js",
-    "setup": "./scripts/setup.sh",
```

Comment:
Why did you removed `npm run setup` ?

Reply:
I actually prefer it removed. Let's not have scripts that we don't run. In a real `setup` for new users, the script actually installs `nodejs` if it doesn't exist. This would be impossible to do if we force users to `npm run setup` because npm wouldn't exist.